redmine_import_time_entry/unit/mapper.py

```diff
@@ -64,11 +66,20 @@ def account_id(self, record):
                 })
 
         project = projects[0]
-
-        return {
+        res = {
             'project_id': project.id,
         }
 
+        if record.get('version'):
+            task = self.env['project.task'].search([
+                ('project_id', '=', project.id),
+                ('name', '=', record['version'])
```

Comment:
Can you describe why to you use versions ?
What about matching with `record['subject']` instead of by versions ?